src/content/docs/identityserver/ui/server-side-sessions/_meta.yml

```diff
@@ -1,3 +1,6 @@
 label: "Server-Side Sessions"
 collapsed: true
-order: 8
\ No newline at end of file
+order: 8
+badge:
+  text: v6.1
+  variant: tip
```

Comment:
```suggestion
```

Bit of an older feature already, would probably not add it to the menu here if the feature has been there for a long time. In the page itself is a good idea though. Thoughts welcome though.